homeassistant/components/opnsense/device_tracker.py

```diff
@@ -1,18 +1,19 @@
-"""Device tracker support for OPNSense routers."""
+"""Device tracker support for OPNsense routers."""
 from homeassistant.components.device_tracker import DeviceScanner
-from homeassistant.components.opnsense import CONF_TRACKER_INTERFACE, OPNSENSE_DATA
+
+from .const import CONF_TRACKER_INTERFACE, OPNSENSE_DATA
 
 
 async def async_get_scanner(hass, config, discovery_info=None):
```

Comment:
This should be updated to use the device tracker for config entries

`from homeassistant.components.device_tracker.config_entry import ScannerEntity`

An example can be found in `unifi`